docs/source/tutorials/llama3.rst

```diff
@@ -78,15 +78,15 @@ We can also add command-line overrides as needed, e.g.
         tokenizer.path=<checkpoint_dir>/tokenizer.model \
         checkpointer.output_dir=<checkpoint_dir>
 
-This will load the Llama3-8B-Instruct checkpoint and tokenizer from ``<checkpoint_dir>`` used in the ``tune download`` command above,
+This will load the Llama3-8B-Instruct checkpoint and tokenizer from ``<checkpoint_dir>`` used in the :ref:`tune download <tune_download_label>` command above,
 then save a final checkpoint in the same directory following the original format. For more details on the
 checkpoint formats supported in torchtune, see our :ref:`checkpointing deep-dive <understand_checkpointer>`.
 
 .. note::
-    To see the full set of configurable parameters for this (and other) configs we can use ``tune cp`` to copy (and modify)
-    the default config. ``tune cp`` can be used with recipe scripts too, in case you want to make more custom changes
-    that cannot be achieved by directly modifying existing configurable parameters. For more on ``tune cp`` see the section on
-    :ref:`modifying configs <tune_cp_label>`.
+    To see the full set of configurable parameters for this (and other) configs we can use :ref:`tune cp <tune_cp_cli_label>` to copy (and modify)
+    the default config. :ref:`tune cp <tune_cp_cli_label>` can be used with recipe scripts too, in case you want to make more custom changes
+    that cannot be achieved by directly modifying existing configurable parameters. For more on :ref:`tune cp <tune_cp_cli_label>` see the section on
+    :ref:`modifying configs <tune_cp_label>` in our ":ref:`finetune_llama_label`" tutorial.
```

Comment:
need <> here?

Reply:
just grabs the title if no text provided